systemd/system/boot.automount

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Boot partition Automount Point
+DefaultDependencies=no
+Before=sysinit.target
+ConditionPathExists=/boot
```

Comment:
Since DefaultDependencies is no there isn't any ordering between this and tmpfiles which is what ensures things like /boot exist. I'd say just drop the condition and let the automount unit create /boot is required.

Conditions that would be useful are:
```
ConditionVirtualization=!container
ConditionPathExists=!/usr/.noupdate
```

That way we don't try try to mount /boot on PXE/ISO images or when booting a coreos based container. The same conditions could go in boot.mount as well.